src/gui/optionsdialog.ui

```diff
@@ -1103,6 +1103,16 @@ Manual: Various torrent properties (e.g. save path) must be assigned manually</s
                  </property>
                 </widget>
                </item>
+               <item>
+                <widget class="QCheckBox" name="checkUseCategoryPaths">
+                 <property name="text">
+                  <string>Use Category paths in "Manual" mode</string>
+                 </property>
+                 <property name="toolTip">
+                  <string>Resolve relative "Save Path" against appropriate Category path instead of Default one</string>
```

Comment:
Randomly mixing uppercase and lowercase looks bad for me. I would suggest: `Resolve relative save path against appropriate category path instead of default one`.
IMO "Save Path" and "Default" is clear enough **without** uppercase and double quotes.

Reply:
https://github.com/qbittorrent/qBittorrent/pull/16330#discussion_r795204828